internal/authz/certiticate_url.go

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2020 Doc.ai and/or its affiliates.
```

Comment:
I feel that these functions can be reused into proxy-nsmgr. Can we move this file to sdk?

Reply:
Why are only for NSE callback identification,

Reply:
ok, got it, thanks